frontend/pulse3d/src/components/interactiveAnalysis/InteractiveAnalysisModal.js

```diff
@@ -643,6 +705,30 @@ export default function InteractiveWaveformModal({
       changelogMessage = `Maximum valleys window changed from ${pvWindow.maxValleys.toFixed(
         2
       )} to ${peakValleyWindows[selectedWell].maxValleys.toFixed(2)}`;
+    } else if (isNewValleyY1 && isNewValleyY2) {
+      changelogMessage = `Valley Line moved ${
+        valleyY1[twentyFourPlateDefinition.getIndexFromWellName(selectedWell)] - valleyY1ToCompare
```

Comment:
`twentyFourPlateDefinition.getIndexFromWellName(selectedWell)` is reused a lot, I think it could be separated out somewhere above the if/else

Reply:
Actually, I think this should just be set somewhere in state if it's being called so much. Just set selectedWellIdx as a state value and then only need to call this once whenever the selected well changes.